src/Typewriter/Options.cs

```diff
@@ -45,7 +45,7 @@ public enum GenerationMode
 
     public class Options
     {
-        [Option('l', "language", Default = "CSharp", HelpText = "The target language for the generated code files. The values can be: Android, Java, ObjC, CSharp, PHP, Python, TypeScript, or GraphEndpointList")]
+        [Option('l', "language", Default = "CSharp", HelpText = "The target language for the generated code files. The values can be: Java, ObjC, CSharp, PHP, Python, TypeScript, or GraphEndpointList")]
```

Comment:
related to this change, we need to update the readme so we no longer note this option.